addons/im_livechat/static/src/core/common/thread_actions_patch.js

```diff
@@ -0,0 +1,43 @@
+import { threadActionsRegistry } from "@mail/core/common/thread_actions";
+import { patch } from "@web/core/utils/patch";
+
+patch(threadActionsRegistry.get("invite-people"), {
+    condition(component) {
+        return (
+            component.thread?.model === "discuss.channel" &&
+            (!component.props.chatWindow || component.props.chatWindow.isOpen) &&
+            !component.thread?.composerDisabled
+        );
+    },
+});
+
+patch(threadActionsRegistry.get("notification-settings"), {
```

Comment:
Why do we need to patch all of those? I think it would be fine to keep the default behavior.